mungegithub/github/github.go

```diff
@@ -2066,7 +2066,7 @@ func (obj *MungeObject) MergePR(who string) bool {
 	if config.DryRun {
 		return true
 	}
-	mergeBody := fmt.Sprintf("Automatic merge from %s.", who)
```

Comment:
This should always terminate with a period. I think it would be cleaner to fix this on the submit-queue side by only appending `". " + sq.MergeToMaster` if `sq.MergeToMaster` is not empty.

Reply:
Thanks! Agreed. Will fix this. But we should make sure that we don't configure `merge-to-master-message` terminating with a period, otherwise, the final comment in K/K merged PR will still have two periods.

https://github.com/kubernetes/test-infra/blob/master/mungegithub/submit-queue/deployment/kubernetes/configmap.yaml#L103
```
merge-to-master-message: "If you want to cherry-pick this change to another branch, please follow the instructions <a href=\"https://github.com/kubernetes/community/blob/master/contributors/devel/cherry-picks.md\">here</a>."
```

The period in this message should be removed.

Reply:
Good point. Thats probably worth commenting on in the config option description and/or in a comment in the yaml config itself. "Must not end in a period."